packages/valory/skills/impact_evaluator_abci/fsm_specification.yaml

```diff
@@ -145,13 +145,13 @@ transition_func:
     (TwitterHashtagsCollectionRound, API_ERROR): TwitterHashtagsCollectionRound
     (TwitterHashtagsCollectionRound, DONE): TwitterDecisionMakingRound
     (TwitterHashtagsCollectionRound, DONE_MAX_RETRIES): TwitterDecisionMakingRound
-    (TwitterHashtagsCollectionRound, NO_MAJORITY): TwitterHashtagsCollectionRound
-    (TwitterHashtagsCollectionRound, ROUND_TIMEOUT): TwitterHashtagsCollectionRound
+    (TwitterHashtagsCollectionRound, NO_MAJORITY): TwitterRandomnessRound
+    (TwitterHashtagsCollectionRound, ROUND_TIMEOUT): TwitterRandomnessRound
     (TwitterMentionsCollectionRound, API_ERROR): TwitterMentionsCollectionRound
     (TwitterMentionsCollectionRound, DONE): TwitterDecisionMakingRound
     (TwitterMentionsCollectionRound, DONE_MAX_RETRIES): TwitterDecisionMakingRound
-    (TwitterMentionsCollectionRound, NO_MAJORITY): TwitterMentionsCollectionRound
-    (TwitterMentionsCollectionRound, ROUND_TIMEOUT): TwitterMentionsCollectionRound
+    (TwitterMentionsCollectionRound, NO_MAJORITY): TwitterRandomnessRound
+    (TwitterMentionsCollectionRound, ROUND_TIMEOUT): TwitterRandomnessRound
```

Comment:
We should select new keepers whenever we time out. I found an issue today where the service was stuck in a timeout loop.